### PR TITLE
Add list_all_authors method and update add_game method with user inpu…

### PR DESCRIPTION
**Pull request by saikatgpae**
…t addition.
This PR is created to add authors and list all authors option"6".

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -65,9 +66,16 @@ def list_all_labels
     end
   end
 
+  # List all authors option "6"
   def list_all_authors
-    # add code here
-    puts 'List all authors'
+    puts 'List of all authors'
+    if @authors.length <= 0
+      puts 'No games to show'
```
Comment (SergSkol) [inline app.rb]: Please change "No games.." to "No authors"

File: main.rb [hunk 2]
```diff
@@ -31,6 +31,7 @@ def choose_option(input)
     when '8' then add_book
     when '9' then add_music_album
     when '10' then add_game
+    when '12' then add_author
```
Comment (SergSkol) [inline main.rb]: There is no such method "add_author". Please remove this option '12'

**Review by SergSkol — CHANGES_REQUESTED** [review]
Hi @saikatgpae ,

# Status: Changes Required ♻️
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Highlights✔️
- Linter checks are passing
- Good commit messages
- Proper use of gitflow

### Required Changes ♻️
   Check the comments under the review.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: app.rb (hunk 1 above)
Comment (saikatgpae) [reply app.rb]: Hi @SergSkol,
Thank you for notifying.
Changed the same.

File: main.rb (hunk 2 above)
Comment (saikatgpae) [reply main.rb]: Hi @SergSkol,
Thank you for notifying.
Removed the same.

**Review by SergSkol — APPROVED** [review]
Status: Approved 🎉 🎉 🎉
Hi @saikatgpae ❕

Your task is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

**Pull request merged**